blog/_posts/2018-07-07-first-official-contribution-team-potatoCode.md

```diff
@@ -0,0 +1,38 @@
+---
+title: First Week Wrap Up! Team potatoCode
+layout: post
+created_at: Sat Jul 07 2018
+permalink: blog/2018-07-07-first-official-contribution-team-potatoCode
+current: blog
+author: Alina Leuca and Sabine van der Eijk
+twitter: alinaleuca
+categories:
+- student-posts
+---
+
+We're well on our way to being a fully-functional team, finally! Today Sabine and I met for the first time to code together and get our wits about the project :) Being part-time comes with its perks, but also its drawbacks.
+
+On the one hand, we have time to do our favorite activities: hang out with the dogs, go surfing, and maybe even take a weekend break away from Amsterdam. On the other hand, both of us have part-time jobs we have to worry about and work around our schedule. We are honored to be one of the first part-time guinea pigs though, and are super excited about Stretchly! We've already started work, and are still slowly digging our way through all the code. It's our first time working with Electron as well, so we're learning tons already :)
+
+A bit of what happened this week:
+We had our first call with our mentor Jan on Wednesday, and he is wonderful! He gave us ideas for first issues and functionalities, and has already helped us figure out how to contribute to the project. We're excited for a potential visit from him some time during the program!
+
+We also had our first call with our supervisor Ana Sofia and had a blast! Besides discussing all the oportunities we'll have to meet up in Portugal or Amsterdam, she also helped us a TON to feel welcomed into the program.
+
+On Saturday we got together to work on Stretchly again, and were lucky to have our coach Dan with us the entire day. We managed to get a lot of work done, and especially decipher the app a lot more with his help.
+
+We completed our first little feature -- translation of the app into Dutch, and our first pull request!
+
+<br>
+<span class="color-red">We had our first dinner with all the coaches!</span>
+<br>
+
+This was a huge highlight of our week! It was Sabine's first time meeting Eden, and both of our first time meeting Jack. We got to know each other a bit, and it helps us feeling more comfortable with our coaches. We can only hope it makes the coaches more comfortable with us as well! We hope to do this again (maybe once a month?) during the program.
+
+![image](/img/blog/2018/2018-07-05-potatocode-team.jpg)
+<div class="image-credits">potatoCode team! (Photo: Sabine van der Eijk, Daniel Rigberg, Jack Sacali, Eden Laboda, Alina Leuca)</div>
```

Comment:
@aleuca Sorry, I missed this in my last review but this seems like just the names of people in the photo. Can you add photo credits, something like, 'photo by [device/person]'? We're good to go after that minor change. Thanks.